packages/connectivity/src/scp-cf/jwt.spec.ts

```diff
@@ -1,7 +1,7 @@
 import { IncomingMessage } from 'http';
 import { Socket } from 'net';
 import { unixEOL } from '@sap-cloud-sdk/util';
-import nock = require('nock');
```

Comment:
These changes are good, but next time consider doing them on a separate PR.

Reply:
Agreed. Will try to do better in the future 🙏